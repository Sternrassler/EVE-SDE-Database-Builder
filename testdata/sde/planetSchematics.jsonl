{"schematicID":65,"schematicName":"Bacteria","cycleTime":1800,"typeID":2393}
{"schematicID":66,"schematicName":"Biofuels","cycleTime":1800,"typeID":2396}
{"schematicID":67,"schematicName":"Biomass","cycleTime":1800,"typeID":3779}
