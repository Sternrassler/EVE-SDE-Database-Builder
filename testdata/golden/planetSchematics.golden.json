[
  {
    "schematicID": 65,
    "cycleTime": 1800
  },
  {
    "schematicID": 66,
    "cycleTime": 1800
  },
  {
    "schematicID": 67,
    "cycleTime": 1800
  }
]